src/app/pages/blog/gated-content-dialog/gated-content-dialog.tsx

```diff
@@ -73,16 +73,13 @@ function GatedContentBody({
 function SubjectSelector() {
     const {subjectSnippet: data} = useBlogContext();
     const options = React.useMemo(
-        () =>
-            camelCaseKeys(
```

Comment:
Is this transformation happening somewhere else now?

Reply:
I realized that camelCaseKeys has no effect, since the keys are all known here (`label` and `value`)